util/topgen/merge.py

```diff
@@ -499,12 +499,12 @@ def amend_resets(top):
             log.error("{} missing type field".format(reset["name"]))
             return
 
-        if reset["type"] in ["gen"]:
+        if reset["type"] in ["top"]:
```

Comment:
Do you plan to list the types here? Or always checking one "top"?

Reply:
ah do you mean i should make the types to check for a separate variable that i check against? I think that might be cleaner.

Reply:
Nope I meant, as the list has only one entry inside, i think this is basically `reset["type"] == "top"`. So I asked if you want to something like this later in your mind. `reset["type"] in ["top", "gen"]` etc

Reply:
o i see you're right. Originally i had more than one type, but it is one now. Let me update.